docs/upgrade.md

```diff
@@ -0,0 +1,58 @@
+# Upgrade
+
+Currently the upgrade can be performed just between follow up downstream builds
+e.g. 171 -> 172.
+There is opened [BZ](https://bugzilla.redhat.com/show_bug.cgi?id=1767400) which
+is tracking the issue which prevents us to upgrade to the latest available build.
```

Comment:
Same as above, since the bug is already VERIFIED, is this paragraph still valid?

Reply:
Yep it's resolved.